qa/integration-tests/src/test/java/io/camunda/it/client/DecisionQueryTest.java

```diff
@@ -34,21 +34,27 @@
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 import org.awaitility.Awaitility;
+import org.junit.jupiter.api.AfterAll;
 import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Test;
 
 @ZeebeIntegration
-public class DecisionQueryTest {
+class DecisionQueryTest {
   private static final List<Decision> DEPLOYED_DECISIONS = new ArrayList<>();
   private static final List<DecisionRequirements> DEPLOYED_DECISION_REQUIREMENTS =
       new ArrayList<>();
   private static ZeebeClient zeebeClient;
 
-  @TestZeebe
-  private static TestStandaloneCamunda testStandaloneCamunda = new TestStandaloneCamunda();
+  @TestZeebe(initMethod = "initTestStandaloneCamunda")
+  private static TestStandaloneCamunda testStandaloneCamunda;
+
+  @SuppressWarnings("unused")
+  static void initTestStandaloneCamunda() {
+    testStandaloneCamunda = new TestStandaloneCamunda();
```

Comment:
I haven't tested, but wouldn't it work if we just initialized `testStandaloneCamunda` in `beforeAll` method instead ?

Reply:
No, I already tried it but it does not work.
`ZeebeIntegrationExtension.beforeAll` is invoked before an expects to find a value for the fields annotated with `@TestZeebe`